server/engine/src/main/java/org/eclipse/lsp/cobol/implicitDialects/cics/utility/CICSWebOptionsCheckUtility.java

```diff
@@ -252,13 +252,24 @@ private void checkConverse(CICSParser.Cics_web_converseContext ctx) {
       checkHasIllegalOptions(
           ctx.USERNAMELEN(), "USERNAMELEN without NONE, BASICAUTH or AUTHENTICATE");
     }
+    checkHasAtLeastOneOption(
+        "INTO or SET or TOLENGTH or TOCONTAINER",
+        ctx,
+        ctx.INTO(),
+        ctx.SET(),
+        ctx.TOLENGTH(),
+        ctx.TOCONTAINER());
 
     checkMutuallyExclusiveOptions(
         "INTO, SET or TOCONTAINER", ctx.INTO(), ctx.SET(), ctx.TOCONTAINER());
```

Comment:
I think this is the condition we should be checking here
```suggestion
    checkHasExactlyOneOption(
        "INTO, SET or TOCONTAINER", ctx.INTO(), ctx.SET(), ctx.TOCONTAINER());
```

Reply:
TOLENGTH / TOCONTAINER difference